<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Residentes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonvista_admin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="residentes_view">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>211</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>449</width>
      <height>209</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
